src/main/java/org/example/LessonDao.java

```diff
@@ -60,29 +55,37 @@ public List<Lesson> getAllLessons() throws SQLException {
              ResultSet resultSet = statement.executeQuery(query)) {
 
             while (resultSet.next()) {
-                int lessonId = resultSet.getInt(1);
-                String lessonName = resultSet.getString(2);
-                int homeworkId = resultSet.getInt(3);
-                String homeworkName = resultSet.getString(4);
-                String homeworkDescription = resultSet.getString(5);
+                int lessonIdIndex = 1;
+                int lessonNameIndex = 2;
+                int homeworkIdIndex = 3;
+                int homeworkNameIndex = 4;
+                int homeworkDescriptionIndex = 5;
```

Comment:
Create 1 variable `index`, and set its value to 1. For the next column just increment it and pass it to the resultSet as follows: `resultSet.getInt(index++)`.